plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -450,17 +440,18 @@ func (r *ResourceFit) PodFitsResources(pod *api.Pod, existingPods []*api.Pod, no
 		return true, nil
 	}
 
-	pods := append(existingPods, pod)
-	_, exceedingCPU, exceedingMemory := CheckPodsExceedingFreeResources(pods, allocatable)
-	if len(exceedingCPU) > 0 {
-		return false, newInsufficientResourceError(cpuResourceName, podRequest.milliCPU,
-			getTotalResourceRequest(existingPods).milliCPU, allocatable.Cpu().MilliValue())
+	totalMilliCPU := allocatable.Cpu().MilliValue()
+	totalMemory := allocatable.Memory().Value()
+	if totalMilliCPU != 0 && totalMilliCPU < podRequest.milliCPU+nodeInfo.RequestedResource().MilliCPU {
```

Comment:
It seems like the logic has been recently changed for 0 values: https://github.com/kubernetes/kubernetes/pull/19890.

Will update that part.